### ISSUE TITLE
[Error Output] Lines available, but no instructions info 

### ISSUE THREAD

**Issue by mgroth0** [Kotlin/kotlinx-kover#447]
**Describe the bug**

I am seeing error output that does not throw any exception and that does not prevent a successful build. It is unclear how I should address this error output.

I tried to use the `annotatedBy` method in `excludes` as a workaround, but it did not work as expected.

**Errors**

I see output like this, just replacing the classname with a real one and replacing X with a small number (I see a number between 1 to 4)

`(Coverage): Class com.example.MyKotlinFileKt does not have instructions info, while there are X lines available: java.lang.Throwable`

**Expected behavior**

[Error Output Issue]: Either there should be no error output at all, or this error output should be more informative and clearly inform the user what action they are supposed to take in order to address the issue.

[`annotatedBy` Issue]: I should be able to exclude the problematic functions or files with annotations.

**Reproducer**

Its not feasible to create a full reproducer project right now. However, I can copy and paste the full contents of one such file where this error output is occuring: 

```kotlin
package my.package

import org.gradle.api.Task
import org.gradle.api.tasks.TaskContainer
import org.gradle.api.tasks.TaskProvider


inline fun <reified T : Task> TaskContainer.register(
    name: String,
    crossinline cfg: T.() -> Unit = {}
): TaskProvider<T> = register(name, T::class.java) {
    cfg.invoke(it)
}
```

Here I write a function `register`. This is pretty much an exact copy of the function from the kotlin gradle dsl. I copied it from the gradle kotlin dsl for a module that uses the gradle api, but does not itself depend on the gradle-kotlin-dsl api.  

**Failed Workaround**

Note also that I tried to get around this by excluding the class with an annotation. I did:

```kotlin
@file:ExcludeFromKover()
package matt.kbuild.glang.mykdsl.regtask.notstupid

package my.package
import org.gradle.api.Task
import org.gradle.api.tasks.TaskContainer
import org.gradle.api.tasks.TaskProvider

@ExcludeFromKover()
inline fun <reified T : Task> TaskContainer.register(
    name: String,
    crossinline cfg: T.() -> Unit = {}
): TaskProvider<T> = register(name, T::class.java) {
    cfg.invoke(it)
}
```

And in my gradle buildscript config I did:

```kotlin
filters {
    includes {
        classes("*")
        packages("*")
    }
    excludes {
        annotatedBy(ExcludeFromKover::class.qualifiedName!!)
    }
}
```
 This had no affect at all. It did not help. I also ensured that `ExcludeFromKover` had its retention set to `RUNTIME`.
 
 Changing the above `excludes` to this:
 
 ```kotlin
 excludes {
  packages("my.package")
}
 ```
 
 made the error output go away.
 
 **Note on the source of the error output**
 
 I did a GitHub search for `"does not have instructions info"` in both this repository and IntelliJ Community, and in both cases got no results. Its therefore a very elusive issue in that I can't find any other discussion of it or even the source code that generates the output.
 
I imagine it has to do with the inlining of the code (I suppose that removes the "instructions"). If I recall correctly, removing the `inline` key made the error output go away. However its strange because this issue doesn't exist with most other inline classes or functions.
 

**Environment**
 - Kover Gradle Plugin version: 0.7.3
 - Gradle version: 8.2.1
 - Kotlin project type: Kotlin/Jvm with a dependency on the Gradle API (basically a gradle plugin)
 - Coverage Toolset: Kover
 - OS: MacOS

**Comment by shanshin** [comment]
Hi,
I think it's better to divide this issue into two, because these problems may not be related to each other and be solved in different releases.

**Comment by shanshin** [comment]
@mgroth0, unfortunately, we were not able to reproduce this problem locally, could you provide a small reproducer project?

**Comment by shanshin** [comment]
Logging have been fixed in the `0.7.4` version
